redis-utils-lib/src/cmr/redis_utils/config.clj

```diff
@@ -20,6 +44,10 @@
   "Timeout for connection in ms."
   {:default 3000 :type Long})
 
+(defconfig redis-collection-metadata-timeout-ms
+  "Timeout for connection in ms."
+  {:default 300000 :type Long})
```

Comment:
consider using math to show that this is 5 min, the zeros all run together with my eyes.  :)

Reply:
fixed.